.github/workflows/test.yml

```diff
@@ -12,7 +12,7 @@ jobs:
     name: Unit tests
     strategy:
       matrix:
-        version: [1.16.x]
+        version: [1.17.x]
```

Comment:
Since this is an application rather than I library I'm inclined to just support one version of Go. Release binaries are available and I want to discourage anyone from using an older, unsupported version of Go to compile.

Reply:
oh yeah, you have a point here!